src/main/lrsql/spec/config.clj

```diff
@@ -228,6 +228,7 @@
                     ::jwt-no-val-role
                     ::jwt-no-val-role-key
                     ::jwt-no-val-logout-url
+                    ::jwt-common-secret
```

Comment:
but it doesn't exist as a def?

Reply:
Yeah you need to add this to the config spec, you'll see all this other stuff in there